README.adoc

```diff
@@ -1,3 +1,45 @@
-= WIP presentation tools
+= ensuite-present
 
-This project is a Work In Progress
+_ensuite-present_ is part of the *ensuite* toolbelt dedicated to trainers and speakers.
+
+== Overview
+
+_ensuite-present_ is a local Web app that allows an instructor to "open" and control a slide-deck.
+It provides a dual screen setup using two browser windows, one for the presentation console that the instructor sees and one for the projector viewport that the class sees.
+
+A screenshot of the presentation console is shown below.
+The left-hand pane of the presentation console shows the current and next slide.
+The right-hand pane shows the speaker notes, with the notes for the current slide highlighted.
+
+image::docs/presentation-console-screenshot.jpg[Screenshot of presentation console]
+
+The projector viewport window shows the current slide fit to the browser window.
+It synchronizes when the instructor advances in the presentation console window.
+
+== Prerequisite
+
+You need https://nodejs.org[Node] 7.4.0 (and npm){blank}footnoteref:[nvm,We strongly recommend using https://github.com/creationix/nvm[nvm] to manage Node.] in order to install and run _ensuite-present_.
+
+ $ nvm install stable
+
+== Installation
+
+_ensuite-present_ is a CLI tool that you install globally with npm.
+
+ $ npm install -g ensuite-present
+
+== Usage
+
+To run _ensuite-present_ on your machine, you just need to run this command:
+
+ $ prz
```

Comment:
I'm happy to defer. My brain is really getting attached to "enpres" though. But no need to rush it. We have time to let it soak.